spec/models/batch_context_spec.rb

```diff
@@ -20,6 +20,10 @@
       expect { bc.user = nil }.to change(bc, :valid?).to(false)
     end
 
+    it 'is not valid if user does not select a file manifest' do
+      expect { bc.using_file_manifest = nil }.to change(bc, :valid?).to(false)
```

Comment:
Isn't this test for a simple image?  When a file manifest is NOT required?

Reply:
yes, good point ... i made the test more accurate and added another